index.html

```diff
@@ -252,41 +252,73 @@ <h3 class="connect-text">LET'S CONNECT</h3>
             <li class="lang-exp">
               <div class="no-1">
                 <p>Languages</p>
-                <img src="images/arrow_bottom.png" alt="arrow_bottom">
+                <img src="images/arrowdown.svg" alt="arrow_bottom" />
               </div>
               <ul class="languages">
                 <li>
-                  <img src="images/js_logo.png" alt="JavaScript" />
+                  <img src="images/javascript.svg" alt="JavaScript" />
                   <a href="#">JavaScript</a>
                 </li>
                 <li>
-                  <img src="images/htmlLogo.png" alt="CSS" />
+                  <img src="images/html.svg" alt="CSS" />
                   <a href="#">HTML</a>
                 </li>
                 <li>
-                  <img src="images/css_logo.png" alt="CSS" />
+                  <img src="images/css.svg" alt="CSS" />
                   <a href="#">CSS</a>
                 </li>
               </ul>
             </li>
             <li class="lang-exp">
               <div class="no-1">
                 <p>Frameworks</p>
-                <img src="images/arrow_bottom.png" alt="arrow_bottom">
+                <img src="images/arrowside.svg" alt="arrow_bottom" />
               </div>
-              <hr>
+              <hr />
             </li>
-          
+
             <li class="lang-exp">
               <div class="no-1">
                 <p>Skills</p>
-                <img src="images/arrow_bottom.png" alt="arrow_bottom">
+                <img src="images/arrowside.svg" alt="arrow_bottom" />
               </div>
-              <hr>
+              <hr />
             </li>
           </ul>
         </div>
       </div>
     </section>
+
+    <!-- Contact Section -->
+    <section class="contact-section">
+      <div class="contact-container">
+        <h2 class="contact-header">Contact Me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form-inputs">
```

Comment:
Hi Great job.
- [ ] As mentioned in the requirements, the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)